packages/utils/index.js

```diff
@@ -1,4 +1,9 @@
+const didSetup = new Set;
+
 function setupBabelPlugins(addon, options) {
+  if (didSetup.has(addon)) return;
```

Comment:
This seems reasonable to me, I feel like we normally store this state on the addon (`addon._didSetupBabel`) but given that we're running this on parent's as well as the child I think it makes sense to use a `Set` here.

Reply:
FWIW, if somebody else stumbles across this, another potential problem with the `addon._didSetupBabel` pattern is that it only works for _one_ addon, that configures the host `includer`. If the host depends on two addons that both setup babel and happen to use the same marker name, one will not run.

Besides using a `Set`, in some cases it might be more appropriate to actually check whether or not certain plugins are included using [`ember-cli-babel-plugin-helpers`](https://github.com/dfreeman/ember-cli-babel-plugin-helpers).